### DISCUSSION TITLE
Workspace restriction

### DISCUSSION THREAD

**Discussion by jkramps** [gwleuverink/clickup-time-tracker#9] [Ideas]
It looks like we can select tasks from the Gedachtegoed bord as well. Sometimes tasks over the workspaces have matching names, which will lead to time tracked on a wrong workspace.

Options:
- Use the env file to restrict workspaces
- Maybe display a small workspace name behind tasks when searching
- ?

**Comment by gwleuverink** [comment]
Good point. I can easily restrict the tasks to those from our own company board, but in some cases it could be nice to have both.

I'm open to both suggestions. Maybe we should take a vote in the slack channel?
